### PR TITLE
Added and changed some Android commands

### PR DESCRIPTION
**Pull request by konimex**
Included:
- Added model support for Android
- Changed CPU command to use native Android ones

### CODE REVIEW

File: neofetch [hunk 1]
```diff
@@ -41,7 +41,10 @@ getos() {
 getmodel() {
     case "$os" in
         "Linux")
-            if [ -f /sys/devices/virtual/dmi/id/product_name ] ||\
+            if [ -d "/system/app/" ] && [ -d "/system/priv-app" ]; then
+                model="$(getprop ro.product.brand) $(getprop ro.product.model)"
```
Comment (dylanaraps) [inline neofetch]: Is it possible to combine these two commands?

```sh
$(getprop ro.product.brand ro.product.model)
```

File: neofetch [hunk 2]
```diff
@@ -640,7 +643,12 @@ getcpu() {
     case "$os" in
         "Linux" | "Windows")
             # Get cpu name
-            cpu="$(awk -F ': | @' '/model name|Processor/ {printf $2; exit}' /proc/cpuinfo)"
+            case "$distro" in
```
Comment (dylanaraps) [inline neofetch]: Change the case to this:

```sh
                "Android"*) cpu="$(getprop ro.product.board)" ;;
                *) cpu="$(awk -F ': | @' '/model name|Processor/ {printf $2; exit}' /proc/cpuinfo)" ;;
```

or

```sh
                "Android"*)
                    cpu="$(getprop ro.product.board)" 
                ;;

                *)
                    cpu="$(awk -F ': | @' '/model name|Processor/ {printf $2; exit}' /proc/cpuinfo)"
                ;;
```

**Comment by dylanaraps** [comment]
Apart from the comments, LGTM.

File: neofetch (hunk 1 above)
Comment (konimex) [inline neofetch]: Unfortunately, no. `getprop` doesn't support it. We can read those two directly from `/system/build.prop`, though.

Comment (dylanaraps) [inline neofetch]: Alright, we'll just leave it like this then. :+1:

**Pull request merged**